### PR TITLE
Remove references to Tuinity

### PR DESCRIPTION
**Pull request by fredster33**
I edited the old site as well, not sure if that's fine. If it's not, please tell me.

### CODE REVIEW

**Review by theomega24 — APPROVED** [review]

File: old/index.php [hunk 1]
```diff
@@ -162,7 +162,6 @@
       <div class="wrapper">
         <ul>
           <li><img src="/images/thx_paper.png" alt="Paper" title="Paper" /></li>
-          <li><img src="/images/thx_tuinity.png" alt="Tuinity" title="Tuinity" /></li>
```
Comment (granny) [inline old/index.php:165]: delete the `thx_tuinity.png` image as well

File: old/index.php [hunk 2]
```diff
@@ -60,7 +60,7 @@
               <li><img src="/images/starlight_th.png" alt="Starlight performance comparison graph" /></li>
               <li>
                 <p>Purpur is built on top of the leading performance mods in the industry.</p>
-                <p>Things like the blazing fast chunk performance thanks to Tuinity's complete rewrite of the light engine, Starlight.</p>
+                <p>Things like the blazing fast chunk performance thanks to Paper's complete rewrite of the light engine, Starlight.</p>
```
Comment (granny) [inline old/index.php:63]: i’d prefer to replace "Paper" with "SpottedLeaf" since, to my knowledge, all the work was done by him.

**Review by granny — CHANGES_REQUESTED** [review]
Kinda didn’t see a reason to update these, which is why I skipped them (not against merging this PR, however).

File: old/index.php (hunk 1 above)
Comment (fredster33) [reply old/index.php:165]: Done.

File: old/index.php (hunk 2 above)
Comment (fredster33) [reply old/index.php]: Done.

**Review by granny — APPROVED** [review]

**Pull request merged**